### ISSUE TITLE
Spatial indexes are dropped on container reboot

### ISSUE THREAD

**Issue by rrowlands** [orientechnologies/orientdb#9915]
### OrientDB Version: 3.2.13 (from docker hub)  

When having a database with a spatial index, if the v3.2 OrientDB Docker container is rebooted the spatial index gets dropped. These same exact indexes work just fine on OrientDB v3.0. Please see stack trace and attached logs.

```
2023-01-27 16:36:21:831 SEVER {db=osmre} Error during load of index 'i10223bc155bc43649a8faa12bf659' [OLuceneSpatialIndex]
java.lang.NullPointerException
	at com.orientechnologies.spatial.factory.OSpatialStrategyFactory.createStrategy(OSpatialStrategyFactory.java:48)
	at com.orientechnologies.spatial.engine.OLuceneGeoSpatialIndexEngine.createSpatialStrategy(OLuceneGeoSpatialIndexEngine.java:56)
	at com.orientechnologies.spatial.engine.OLuceneSpatialIndexEngineAbstract.init(OLuceneSpatialIndexEngineAbstract.java:75)
	at com.orientechnologies.spatial.engine.OLuceneSpatialIndexEngineDelegator.init(OLuceneSpatialIndexEngineDelegator.java:85)
	at com.orientechnologies.lucene.index.OLuceneIndexNotUnique.lambda$onIndexEngineChange$3(OLuceneIndexNotUnique.java:296)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.doCallIndexEngine(OAbstractPaginatedStorage.java:3218)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.callIndexEngine(OAbstractPaginatedStorage.java:3197)
	at com.orientechnologies.lucene.index.OLuceneIndexNotUnique.onIndexEngineChange(OLuceneIndexNotUnique.java:291)
	at com.orientechnologies.orient.core.index.OIndexAbstract.loadFromConfiguration(OIndexAbstract.java:324)
	at com.orientechnologies.orient.core.index.OIndexManagerShared.fromStream(OIndexManagerShared.java:1018)
	at com.orientechnologies.orient.core.index.OIndexManagerShared.load(OIndexManagerShared.java:111)
	at com.orientechnologies.orient.core.db.OSharedContextEmbedded.load(OSharedContextEmbedded.java:91)
	at com.orientechnologies.orient.core.db.document.ODatabaseDocumentEmbedded.loadMetadata(ODatabaseDocumentEmbedded.java:336)
	at com.orientechnologies.orient.core.db.document.ODatabaseDocumentEmbedded.init(ODatabaseDocumentEmbedded.java:201)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.open(OrientDBEmbedded.java:503)
	at com.orientechnologies.orient.core.db.OrientDBDistributed.open(OrientDBDistributed.java:254)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.open(OrientDBEmbedded.java:440)
	at com.orientechnologies.orient.core.db.OrientDBDistributed.open(OrientDBDistributed.java:239)
	at com.orientechnologies.orient.server.OServer.openDatabase(OServer.java:842)
	at com.orientechnologies.orient.server.OServer.openDatabase(OServer.java:835)
	at com.orientechnologies.orient.server.network.protocol.http.command.OServerCommandAuthenticatedDbAbstract.authenticate(OServerCommandAuthenticatedDbAbstract.java:194)
	at com.orientechnologies.orient.server.network.protocol.http.command.OServerCommandAuthenticatedDbAbstract.beforeExecute(OServerCommandAuthenticatedDbAbstract.java:136)
	at com.orientechnologies.orient.server.network.protocol.http.command.get.OServerCommandGetConnect.beforeExecute(OServerCommandGetConnect.java:55)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:239)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:800)
	at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:68)

```

My index is defined as:

![image](https://user-images.githubusercontent.com/1326328/215152543-d54f20e6-84ff-45d0-a2aa-1c09b49b4ee9.png)

Full logs:

[orientdb-spatial-drop.txt](https://github.com/orientechnologies/orientdb/files/10521371/orientdb-spatial-drop.txt)

**Comment by tglman** [comment]
Hi,

This should be fixed in the 3.2.16 just released, could you please verify it ?

Regards

**Comment by rrowlands** [comment]
Working for me. Thanks
